src/test/java/org/cloudfoundry/promregator/MonoErrorHandlingTest.java

```diff
@@ -3,16 +3,16 @@
 import java.time.Duration;
 import java.time.Instant;
 
-import org.junit.Assert;
-import org.junit.Test;
 
+import org.junit.jupiter.api.Test;
 import reactor.core.publisher.Mono;
 
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
 public class MonoErrorHandlingTest {
```

Comment:
minor: 😀 This class was once a playground for Mono's, because I did not trust the reactorProject properly... 
Perhaps we can even think about getting rid of it completely...

Reply:
I've deleted it now